appveyor.yml

```diff
@@ -1,30 +0,0 @@
-# appveyor file
-# http://www.appveyor.com/docs/appveyor-yml
```

Comment:
Appveyor would test this repo on a Windows VM, which could be useful to catch any issues with paths (forward slash vs backslash). I see that such tests were run on the last merged PR, a year ago:
https://ci.appveyor.com/project/resin-io/resin-settings-client/builds/19762446/job/7i505crvtg55wqfj

I don't see any similar tests being run for this PR, which could be because this file was deleted (or because the GitHub repo settings were otherwise configured?).

Reply:
That's a good point, I'll re-add the Appveyor tests!